### PR TITLE
fix: some errors in the GitHub templates

### PR DESCRIPTION
**Pull request by raphj**
Related to #2982

## Checkpoints

**For code reviewer**

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

**For developer**

- [X] Manual tests done to ensure stability on whole application layers, issue expectation follow
    Not relevant
- [x] Original issue have been updated with a short resume of implemented solution

**For quality team**

- [x] Tested by quality team

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -72,4 +72,4 @@ assignees: ""
 
 ## Extra information
 
-*Optional - add any extra information which will be usefull for understanding and developing the feature*
+*Optional - add any useful information to understand and develop the feature*
```
Comment (PhilippeAccorsi) [inline .github/ISSUE_TEMPLATE/bug_report.md:75]: Why not change title also ?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -37,4 +37,4 @@ assignees: ""
 
 ## Extra information
 
-*Optional - Add any extra information which will be usefull for understanding and developing the feature*
+*Optional - Add any useful information to understand and develop the feature*
```
Comment (PhilippeAccorsi) [inline .github/ISSUE_TEMPLATE/feature_request.md:40]: Why not change title also ?

Comment (raphj) [reply .github/ISSUE_TEMPLATE/feature_request.md:40]: This was just a simplification of this sentence, to also avoir repeating the title which still seems good as-is. Do you agree with this?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (raphj) [reply .github/ISSUE_TEMPLATE/bug_report.md:75]: See next discussion :-)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply .github/ISSUE_TEMPLATE/feature_request.md:40]: Ok for me.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply .github/ISSUE_TEMPLATE/bug_report.md:75]: Ok for me.

**Comment by PhilippeAccorsi** [comment]
No test necessary for this fix.
Verified.

**Pull request merged**